### PR TITLE
Created analyser for France’s ONF data on public forests

### PR DESCRIPTION
**Pull request by StC-OSM**

### CODE REVIEW

File: analysers/analyser_merge_forest_compartment_FR.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            'parcelles forestières',
+            SHP(Source(
+                attribution='Office National des Forêts',
+                millesime='2023-03-12',
+                fileUrl='http://ws.carmencarto.fr/WFS/105/ONF_Forets?request=GetFeature&service=WFS&version=1.1.0&typeName=PARC_PUBL_FR&outputFormat=SHAPE'),
+                zip='PARC_PUBL_FR.shp'),
+            Load_XY(('ST_X(ST_PointOnSurface(ST_GeometryN(geom, 1)))',), ('ST_Y(ST_PointOnSurface(ST_GeometryN(geom, 1)))',)),
+            Conflate(
+                select = Select(
+                    types = ['nodes','ways','relations'],
```
Comment (frodrigo) [inline analysers/analyser_merge_forest_compartment_FR.py:46]:
```suggestion
                    types = ['ways', 'relations'],
```

File: analysers/analyser_merge_forest_compartment_FR.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+        self.init(
+            'http://metadata.carmencarto.fr/geonetwork/105/api/records/fr-662043116-82880F0D-E1C4-4EF3-80AF-416977F118F1',
+            'parcelles forestières',
+            SHP(Source(
+                attribution='Office National des Forêts',
+                millesime='2023-03-12',
+                fileUrl='http://ws.carmencarto.fr/WFS/105/ONF_Forets?request=GetFeature&service=WFS&version=1.1.0&typeName=PARC_PUBL_FR&outputFormat=SHAPE'),
+                zip='PARC_PUBL_FR.shp'),
+            Load_XY(('ST_X(ST_PointOnSurface(ST_GeometryN(geom, 1)))',), ('ST_Y(ST_PointOnSurface(ST_GeometryN(geom, 1)))',)),
```
Comment (frodrigo) [inline analysers/analyser_merge_forest_compartment_FR.py:43]: Why need of `ST_GeometryN()` ?

File: analysers/analyser_merge_forest_compartment_FR.py (hunk 1 above)
Comment (frodrigo) [reply analysers/analyser_merge_forest_compartment_FR.py:46]: We may use `'polygon'` in place of `'ways','relations'`. There is already a partial implementation.

File: analysers/analyser_merge_forest_compartment_FR.py (hunk 2 above)
Comment (StC-OSM) [reply analysers/analyser_merge_forest_compartment_FR.py:43]: I was not sure if there were multipolygons in the open data. In any case, this is at the edge of my current skills and it was safer for me to keep what was in the example I reused and that appeared to work for me as well.

File: analysers/analyser_merge_forest_compartment_FR.py (hunk 1 above)
Comment (StC-OSM) [reply analysers/analyser_merge_forest_compartment_FR.py:46]: Unfortunately, there are some forest compartments implemented in OSM as nodes.

**Comment by frodrigo** [comment]
You should the analyzer to the config.py file

File: analysers/analyser_merge_forest_compartment_FR.py (hunk 2 above)
Comment (frodrigo) [inline analysers/analyser_merge_forest_compartment_FR.py:43]:
```suggestion
            Load_XY(('ST_X(ST_PointOnSurface(geom))',), ('ST_Y(ST_PointOnSurface(geom))',)),
```

File: analysers/analyser_merge_forest_compartment_FR.py (hunk 1 above)
Comment (frodrigo) [reply analysers/analyser_merge_forest_compartment_FR.py:46]: Support of `polygons` implemented here #2255 and here #2256.

**Comment by frodrigo** [comment]
Closed in favor of #2256

**Pull request closed without merging**